PrestaShop/PrestaShop
# Add the language and the shop concerned in the Admin logs controller

| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop 
| Description?  | The indicators present in the logs are not complete enough to allow us to understand the origin of certain problems, mainly when we are in a multiboutique and multi-lang context. It will be interesting for example to know in which language and shop the action was made on the object. It's a little more verbose I think.
| Type?         | improvement 
| Category?     | BO 
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #18340
| How to test?  | In a multiboutique context:  
 - Edit a product
- Go to the log controller
- Check in which context (language and shop) the change was made ... there is no!




---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/prestashop/prestashop/16425)
